modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -278,16 +278,15 @@ export class Keychains implements IKeychains {
    * @return {Promise<KeychainsTriplet>} newly created User, Backup, and BitGo keys
    */
   async createMpc(params: CreateMpcOptions): Promise<KeychainsTriplet> {
-    if (_.isUndefined(params.passphrase)) {
-      throw new Error('missing required param passphrase');
-    }
-
     let MpcUtils;
     switch (params.multisigType) {
       case 'tss':
         MpcUtils = this.baseCoin.getMPCAlgorithm() === 'ecdsa' ? ECDSAUtils.EcdsaUtils : EDDSAUtils.default;
         break;
       case 'blsdkg':
+        if (_.isUndefined(params.passphrase)) {
```

Comment:
Do we need to throw this one more time after the `switch` for line `297` since that is normally required?

Reply:
Good point, will make that change